README.md

```diff
@@ -15,242 +15,22 @@ implemented.
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `develop`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `flake`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `help`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `profile`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `repl`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `run`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `search`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `shell`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `bundle`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `copy`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `edit`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `log`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `path-info`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `registry`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `why-depends`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `daemon`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `describe-stores`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `key`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `nar`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `print-dev-env`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `realisation`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `show-config`
-
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
-
-- 🌑 `store`
+- 🌕 stage 0: evaluate basic expressions, rec attrsets, let bindings, `with`
+  statement, functions
 
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
+- 🌕 stage 1: lazy evaluation
 
-- 🌑 `doctor`
+- 🌘 stage 2:
 
-  - 🌑 stage 1 (MVP)
-  - 🌑 stage 2: most common use cases
-  - 🌑 stage 3: full implementation
+  - 🌘 built-in functions (progress: 3 out of 111)
```

Comment:
Oh, we can update this:
```suggestion
  - 🌘 built-in functions (progress: 7 out of 113)
```

Hah, nix has added a couple of new builtins since the last time I checked.